test/duplicate_test.go

```diff
@@ -29,7 +29,7 @@ import (
 	knativetest "knative.dev/pkg/test"
 )
 
-// TestDuplicatePodTaskRun creates 10 builds and checks that each of them has only one build pod.
+// TestDuplicatePodTaskRun creates 5 builds and checks that each of them has only one build pod.
```

Comment:
nit: This should say "TaskRuns", and not "builds"

Reply:
it shouldn't say "build pod" either